src/period-calculation/generate-fixed-periods/generate-fixed-periods-daily.ts

```diff
@@ -32,7 +32,10 @@ const generateFixedPeriodsDaily: GenerateFixedPeriodsDaily = ({
     for (let i = 0; i < day.daysInYear; i++) {
         const nextDay = day.add({ days: i })
 
-        if (endsBefore && Temporal.PlainDate.compare(nextDay, endsBefore) < 1) {
+        if (
+            endsBefore &&
+            Temporal.PlainDate.compare(nextDay, endsBefore) > -1
```

Comment:
it might be a naming issue, but in isolation, I would think that this should be `Temporal.PlainDate.compare(nextDay, endBefore) > 0` ? Like now it is not breaking if nextDay and endBefore are the same...but by the name endBefore, I would think nextDay has to be before endBefore?

Reply:
> but by the name endBefore, I would think nextDay has to be before endBefore?

Yes, that is the case now. When
* nextDay < endsBefore, then `compare` will yield `-1`
* nextDay = endsBefore, then `compare` will yield `0`
* nextDay > endsBefore, then `compare` will yield `1`

So with the current implementation, it works as expected (confirmed by the test)